src/test/rustdoc/intra-doc-link-generic-params.rs

```diff
@@ -0,0 +1,10 @@
+#![crate_name = "foo"]
+
+//! Here's a link to [`Vec<T>`] and one to [`Box<Vec<Option<T>>>`].
+//!
+//! And what about a link to just [`Option`] and, with the generic, [`Option<T>`]?
+
+// @has foo/index.html '//a/@href' '../alloc/vec/struct.Vec.html'
+// @has foo/index.html '//a/@href' '../alloc/boxed/struct.Box.html'
+// @has foo/index.html '//a/@href' '../core/option/enum.Option.html'
+// @has foo/index.html '//a/@href' '../core/option/enum.Option.html'
```

Comment:
Hmm, this test is failing:

```
stdout:
------------------------------------------

------------------------------------------
stderr:
------------------------------------------
7: @has check failed
	`XPATH PATTERN` did not match
	// @has foo/index.html '//a/@href' '../alloc/vec/struct.Vec.html'
8: @has check failed
	`XPATH PATTERN` did not match
	// @has foo/index.html '//a/@href' '../alloc/boxed/struct.Box.html'
9: @has check failed
	`XPATH PATTERN` did not match
	// @has foo/index.html '//a/@href' '../core/option/enum.Option.html'
10: @has check failed
	`XPATH PATTERN` did not match
	// @has foo/index.html '//a/@href' '../core/option/enum.Option.html'

Encountered 4 errors

------------------------------------------
```

Is there any way to see what the actual page was?

Reply:
Oops, forgot about `./x.py doc --stage 1` 😄

Reply:
Actually that doesn't work.

Reply:
An easier way is to use `rustdoc +stage1`: https://rustc-dev-guide.rust-lang.org/building/how-to-build-and-run.html?highlight=rustup,to#creating-a-rustup-toolchain